todolist/ContentView.swift

```diff
@@ -11,24 +11,26 @@ import SwiftUI
 struct TodoItem: Identifiable {
   let id: Int
   let title: String
+  let accessibilityLabel: String
   var done: Bool = false
 }
 
 struct ContentView: View {
   @State var todos = [
-     TodoItem(id: 1, title: "Sample 1"),
-     TodoItem(id: 2, title: "Sample 2"),
-     TodoItem(id: 3, title: "Sample 3")
+     TodoItem(id: 1, title: "Buy more baldo", accessibilityLabel: "Buy more baldo"),
+     TodoItem(id: 2, title: "Invite GuriC to after",
+              accessibilityLabel: "Invite guriso to after"),
```

Comment:
typo in `gurise` (?) 😂

Reply:
this was the closest pronunciation. Closing this PR since its not passing the no-bullying compliance